src/chapter_2/installation.md

```diff
@@ -35,4 +36,26 @@ Now verify the CLI is installed properly.
 
 ```
 anchor --version
-```
\ No newline at end of file
+```
+
+### Installing using Anchor version manager (avm)
+
+Anchor version manager is a tool for using multiple versions of the anchor-cli. It will require the same dependencies as building from source. It is recommended you uninstall the NPM package if you have it installed.
+
+Install `avm` using Cargo. Note this will replace your `anchor` binary if you had one installed.
+
+```
+cargo install --git https://github.com/project-serum/anchor --locked --force
+```
```

Comment:
this doesnt work on my machine.
```
error: multiple packages with binaries found: anchor-cli, avm, example.
When installing a git repository, cargo will always search the entire repo for any Cargo.toml.
Please specify which to install.
```

Im thinking the `avm` is missing? That worked for me
```
cargo install --git https://github.com/project-serum/anchor avm --locked --force
```

Reply:
Oops good spot, thanks, yep `avm` is missing.